kotlin/workflow-ui-android/src/main/java/com/squareup/workflow/ui/WorkflowViewStub.kt

```diff
@@ -79,9 +78,9 @@ class WorkflowViewStub @JvmOverloads constructor(
         }
 
     return when (val parent = actual.parent) {
-      is ViewGroup -> registry.buildView(rendering, containerHints, parent)
+      is ViewGroup -> containerHints[ViewRegistry].buildView(rendering, containerHints, parent)
```

Comment:
Could we provide a top-level `buildView` function that would have the same arguments, and just pull the `ViewRegistry` out of the hints itself? Would clean up all these call sites a little.

Reply:
Yeah, I like that.

Reply:
Meh. Interferes with the convention of having companion objects implementing `ViewBinding` — the `buildView` function there takes precedent. And it's not all that much of a win, because most custom containers can use `WorkflowViewStub` instead of calling `buildView` themselves.

Reply:
The other option is `ContainerHints.buildView` but that feels very wrong.

Reply:
Yeah, I went through the same thought process.

It's fine, I don't think a lot of people are going to write this fragment. The main win from this PR is the ability to use `WorkflowViewStub` without having to figure out how to get your hands on a `ViewRegistry`. I'm content.